portfolio/src/main/webapp/script.js

```diff
@@ -12,23 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/** slideIndex is a global variable */ 
 var slideIndex = 1;
 
-// Next/previous controls
+/** Next/previous controls */
```

Comment:
Nit comment on method comment - "Flips slides to the previous or next page."

https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments
Method descriptions begin with a verb phrase that describes what the method does. This phrase is not an imperative sentence, but instead is written in the third person, as if there is an implied This method ... before it.

Reply:
No need to change every occurrence in this PR, but please adopt this practice in future projects :)